pkgs/development/libraries/cfitsio/default.nix

```
@@ -2,22 +2,25 @@
 
 # Optional dependencies
 , bzip2 ? null }:
-
-stdenv.mkDerivation rec {
+let
+  trimmedVersion = "3.47";
+in stdenv.mkDerivation rec {
   name = "cfitsio-${version}";
```

nitpick:
```suggestion
  pname = "cfitsio";
```


@xbreak what do you think about using just `3.47` for the version?

---

Thanks, will fix that.

Regarding the version number it would be nice to go for the 3.47 since it seems to be the one exclusively used in the library itself (homepage is inconsistent). If it wasn't for the Nixpkgs history of using the long versions I'd say yes definitely. The downside to if we change is that the new package version may look like a downgrade (do we care?). See also other distros on [repology](https://repology.org/project/cfitsio/information). 

Please let me know which way to go and I'll update the PR.

---

Version numbering in `cfitsio` is a bit of mess, but they seem to have dropped the extra '0', as the name of the tarball suggests. I would therefore go for 3.47.

---

Ok, I've updated the PR to reflect this. Thanks!